tests/svs/lib/timing.cpp

```diff
@@ -45,6 +45,22 @@ void stress(svs::lib::Timer& timer, size_t max_depth = 3, size_t max_flat = 3) {
         }
     }
 }
+
+/*
+ * We use a busy_sleep here, not std::this_thread::sleep_for
+ * std::this_thread::sleep_for garanties, that the sleep would be
+ * not shorter than requared.
+ * For some platforms (like MacOS) a sleep can be much longer,
+ * so if we need the sleep time to be close to the requared,
+ * busy sleep is a good choise.
+ */
```

Comment:
```suggestion
/*
 * We use a busy_sleep instead of std::this_thread::sleep_for.
 * std::this_thread::sleep_for guarantees that the sleep will be
 * not shorter than the required time.
 * For some platforms (like MacOS), a sleep can be much longer,
 * so if we need the sleep time to be close to the required,
 * busy sleep is a good choice.
 */
```
Some typos here. Also can you run clang-formating with these commands in the root directory:
sudo apt install clang-format-15 (if clang-format-15 not installed)
bash tools/clang-format.sh clang-format-15